parts/k8s/cloud-init/nodecustomdata.yml

```diff
@@ -246,7 +214,7 @@ write_files:
     {{end}}
 {{end}}
 
-{{if IsNSeriesSKU .}}
+{{if IsNSeriesSKU .VMSize}}
```

Comment:
just to confirm - is this change in AKS RP?

Reply:
No, it is not in RP. Isn’t the point of this PR to vendor AKS engine so we can use it in RP? How would I be able to make the change first there?

Sorry for the confusion :(

Reply:
Yeah, suggest a pair review on this one since this one will be tough to ingest on the current setup. I booked some time tomorrow afternoon to discuss open points them and allow some more discussion until then. 

Thanks for the effort folks

Reply:
nvm. I see the VMSize property in the agent pool.